run/GCClassic/HISTORY.rc.templates/HISTORY.rc.fullchem

```diff
@@ -564,7 +564,7 @@ COLLECTIONS: 'Restart',
 #
 # Archives chemical reaction rates from the KPP solver.
 # It is best to list individual reactions to avoid using too much memory.
-# Reactions should be listed as "RxnRate_EQnnn", where nnn is the reaction
+# Reactions should be listed as "RxnRate_EQnnnn", where nnnn is the reaction
```

Comment:
We would also have to make similar updates in:

- `run/GCClassic/HISTORY.rc.templates/HISTORY.rc.carbon`
- `run/GCClassic/HISTORY.rc.templates/HISTORY.rc.Hg`

as these simulations also use KPP_based mechanisms.

Reply:
These template files currently don't contain `RxnRate` or `RxnConst` collections.